index.js

```diff
@@ -11,15 +11,14 @@ function displayList() {
   books.forEach((item, i) => {
     const bookDiv = document.createElement('div');
     bookDiv.classList.add('book');
-    bookDiv.innerHTML = `
-          <p> Title:${item.title}</p>
-          <p> Author:${item.author}</p>
-          <button data-i="${i}" id="remove">Remove</button>
-          <hr>
+    bookDiv.innerHTML = `<ul>
+          <li>Title:${item.title} By Author:${item.author}   <button data-i="${i}" id="remove">Remove</button> </li>
+          </ul>
         `;
     bookList.appendChild(bookDiv);
   });
 }
+
 // add new book
 function newBook() {
```

Comment:
- [ ] Following the project requirements you are supposed to use classes and arrays but you have used functions please double check the project requirements and update that

![image](https://user-images.githubusercontent.com/75126481/218834900-48401775-a800-4ff2-9829-62263106e946.png)